packages/react-examples/src/react/TimePicker/TimePicker.Example.tsx

```diff
@@ -15,46 +19,72 @@ const timePickerStyles: Partial<IComboBoxStyles> = {
 };
 
 const onFormatDate = (date: Date) => `Custom prefix + ${date.toLocaleTimeString()}`;
-const onChange = (_: React.FormEvent<IComboBox>, date: Date) => console.log('SELECTED DATE: ', date);
+const onTimeChange = (date: Date) => console.log('SELECTED DATE: ', date);
 
 export const TimePickerBasicExample: React.FC = () => {
+  const [datePickerDate, setDatePickerDate] = React.useState<Date>();
+  const [currentTimeString, setCurrentTimeString] = React.useState<string>('');
+
+  const onSelectDate = React.useCallback((selectedDate: Date) => {
+    setDatePickerDate(selectedDate);
+  }, []);
+
+  const onDateTimePickerChange = React.useCallback((time: Date) => {
+    setCurrentTimeString(time.toString());
+  }, []);
+
   const timeRange: ITimeRange = {
     start: 8,
     end: 14,
   };
 
   return (
-    <Stack tokens={stackTokens} styles={stackStyles}>
-      <TimePicker
-        styles={timePickerStyles}
-        useHour12
-        allowFreeform
-        autoComplete="on"
-        label={'TimePicker basic example'}
-        onChange={onChange}
-        defaultValue={new Date('November 25, 2021 09:15:00')}
-        useComboBoxAsMenuWidth
-      />
-      <TimePicker
-        styles={timePickerStyles}
-        showSeconds
-        allowFreeform
-        increments={15}
-        autoComplete="on"
-        label={'TimePicker with non default options'}
-        useComboBoxAsMenuWidth
-        timeRange={timeRange}
-        onChange={onChange}
-      />
-      <TimePicker
-        styles={timePickerStyles}
-        // eslint-disable-next-line react/jsx-no-bind
-        onFormatDate={onFormatDate}
-        useHour12
-        allowFreeform
-        autoComplete="on"
-        label={'TimePicker with custom time string'}
-      />
-    </Stack>
+    <>
+      <Stack tokens={stackTokens} styles={stackStyles}>
+        <TimePicker
+          styles={timePickerStyles}
+          useHour12
+          allowFreeform
+          autoComplete="on"
+          label={'TimePicker basic example'}
+          onTimeChange={onTimeChange}
+          currentDate={new Date('November 25, 2021 09:15:00')}
+          useComboBoxAsMenuWidth
```

Comment:
nit: it would be helpful for comparison if the props in each example where in the same order as much as possible.

For example, `useComboBoxAsMenuWidth` is the last prop here but not in the next example.

Reply:
Resolved by reorganizing TimePicker examples and setting `useComboBoxAsMenuWidth` as fixed TimePicker prop.